filebeat/docs/inputs/input-common-options.asciidoc

```diff
@@ -84,3 +84,8 @@ this option usually results in simpler configuration files. If the pipeline is
 configured both in the input and output, the option from the
 input is used.
 
+[float]
+===== `keep_null`
+
+If this option is set to true, fields with `null` values will be published in
+the output document. By default, `keep_null` is set to `false`.
```

Comment:
Hey @dedemorton, I was wondering if you could review this doc snippet before I copy it to other Beats' docs. Thanks!

Reply:
LGTM.